tutorials/yes-no-game/full-game/yesnogame.edn

```diff
@@ -2,9 +2,9 @@
 (def max-timer 5)
 
 (defshards load-resources []
-  (LoadImage "data/cat01.png") (Push :Name .images)
-  (LoadImage "data/cat02.png") (Push :Name .images)
-  (LoadImage "data/cat03.png") (Push :Name .images))
+  (LoadImage "data/cats/cat01.png") (Push :Name .images)
```

Comment:
Can you move the actual files in this folder as well, since they are still using `data/cat01.png`, etc.

Reply:
Got it, the files have been moved in commit [13ec9b0](https://github.com/fragcolor-xyz/shards-examples/pull/57/commits/13ec9b0c1e8ba90aaf45b212451bee1ee0367c5f). 

The files have also been resized to have a horizontal size of 400 pixels as described in Step 2, so running the tutorial from the repo will not have images that are too large. The original images have been stored in a new folder named "original-images" in case it may still be needed, but I can remove the files entirely if they're not.

Reply:
If they're not used in the scripts you can remove the original ones, if we need them ourselves they'll still be in the git history.